diff --git a/dist/core.js b/dist/core.js
index dbc417b28f09ec9d56e49633cdaa1721964a7dda..2780ed010ff7bbda98e7fedb4bad49619ef42a6d 100644
--- a/dist/core.js
+++ b/dist/core.js
@@ -549,8 +549,10 @@ function checkKeyword(keyword, def) {
     (0, util_1.eachItem)(keyword, (kwd) => {
         if (RULES.keywords[kwd])
             throw new Error(`Keyword ${kwd} is already defined`);
-        if (!KEYWORD_NAME.test(kwd))
-            throw new Error(`Keyword ${kwd} has invalid name`);
+        // commenting out to allow '@context' as a keyword - see apps/site/src/lib/jsonSchema.ts
+        // @todo fix this properly
+        // if (!KEYWORD_NAME.test(kwd))
+        //     throw new Error(`Keyword ${kwd} has invalid name`);
     });
     if (!def)
         return;
