diff --git a/node_modules/@aws-sdk/lib-storage/dist-cjs/Upload.js b/node_modules/@aws-sdk/lib-storage/dist-cjs/Upload.js
index 49de722..9148b94 100644
--- a/node_modules/@aws-sdk/lib-storage/dist-cjs/Upload.js
+++ b/node_modules/@aws-sdk/lib-storage/dist-cjs/Upload.js
@@ -3,6 +3,7 @@ Object.defineProperty(exports, "__esModule", { value: true });
 exports.Upload = void 0;
 const abort_controller_1 = require("@aws-sdk/abort-controller");
 const client_s3_1 = require("@aws-sdk/client-s3");
+const smithy_client = require("@aws-sdk/smithy-client");
 const events_1 = require("events");
 const bytelength_1 = require("./bytelength");
 const chunker_1 = require("./chunker");
@@ -42,8 +43,30 @@ class Upload extends events_1.EventEmitter {
     async __uploadUsingPut(dataPart) {
         this.isMultiPart = false;
         const params = { ...this.params, Body: dataPart.data };
-        const putResult = await this.client.send(new client_s3_1.PutObjectCommand(params));
-        this.putResponse = putResult;
+        // See https://github.com/aws/aws-sdk-js-v3/pull/2700/files#diff-12fa2b5136167522f27fddb10706f410daffd1fef7a7ac5a5cac5b5a273fc588
+        // for code below.
+        // The PR https://github.com/aws/aws-sdk-js-v3/pull/2700 needs to be merged before removing this patch.
+        const [putResult, endpoint] = await Promise.all([
+            this.client.send(new client_s3_1.PutObjectCommand(params)),
+            this.client.config.endpoint(),
+          ]);
+      
+          const locationKey = this.params
+            .Key.split("/")
+            .map((segment) => smithy_client.extendedEncodeURIComponent(segment))
+            .join("/");
+          const locationBucket = smithy_client.extendedEncodeURIComponent(this.params.Bucket);
+      
+          const Location = this.client.config.forcePathStyle
+            ? `${endpoint.protocol}//${endpoint.hostname}/${locationBucket}/${locationKey}`
+            : `${endpoint.protocol}//${locationBucket}.${endpoint.hostname}/${locationKey}`;
+      
+          this.putResponse = {
+            ...putResult,
+            Bucket: this.params.Bucket,
+            Key: this.params.Key,
+            Location,
+          };
         const totalSize = bytelength_1.byteLength(dataPart.data);
         this.__notifyProgress({
             loaded: totalSize,
