diff --git a/node_modules/@changesets/get-dependents-graph/dist/get-dependents-graph.cjs.dev.js b/node_modules/@changesets/get-dependents-graph/dist/get-dependents-graph.cjs.dev.js
index 9bd1b9d..cbacc42 100644
--- a/node_modules/@changesets/get-dependents-graph/dist/get-dependents-graph.cjs.dev.js
+++ b/node_modules/@changesets/get-dependents-graph/dist/get-dependents-graph.cjs.dev.js
@@ -71,6 +71,9 @@ function getDependencyGraph(packages, opts) {
     for (let [depName, depRange] of allDependencies) {
       const match = packagesByName[depName];
       if (!match) continue;
+      if (match.packageJson.private) {
+        continue;
+      }
       const expected = match.packageJson.version;
       const usesWorkspaceRange = depRange.startsWith("workspace:");
 
diff --git a/node_modules/@changesets/get-dependents-graph/dist/get-dependents-graph.cjs.prod.js b/node_modules/@changesets/get-dependents-graph/dist/get-dependents-graph.cjs.prod.js
index b4026bb..ef99cd5 100644
--- a/node_modules/@changesets/get-dependents-graph/dist/get-dependents-graph.cjs.prod.js
+++ b/node_modules/@changesets/get-dependents-graph/dist/get-dependents-graph.cjs.prod.js
@@ -45,6 +45,9 @@ function getDependencyGraph(packages, opts) {
     for (let [depName, depRange] of allDependencies) {
       const match = packagesByName[depName];
       if (!match) continue;
+      if (match.packageJson.private) {
+        continue;
+      }
       const expected = match.packageJson.version;
       if (depRange.startsWith("workspace:")) {
         if (depRange = depRange.replace(/^workspace:/, ""), "*" === depRange || "^" === depRange || "~" === depRange) {
diff --git a/node_modules/@changesets/get-dependents-graph/dist/get-dependents-graph.esm.js b/node_modules/@changesets/get-dependents-graph/dist/get-dependents-graph.esm.js
index b0eaa77..98f3526 100644
--- a/node_modules/@changesets/get-dependents-graph/dist/get-dependents-graph.esm.js
+++ b/node_modules/@changesets/get-dependents-graph/dist/get-dependents-graph.esm.js
@@ -62,6 +62,9 @@ function getDependencyGraph(packages, opts) {
     for (let [depName, depRange] of allDependencies) {
       const match = packagesByName[depName];
       if (!match) continue;
+      if (match.packageJson.private) {
+        continue;
+      }
       const expected = match.packageJson.version;
       const usesWorkspaceRange = depRange.startsWith("workspace:");
 
