diff --git a/node_modules/@changesets/cli/dist/cli.cjs.dev.js b/node_modules/@changesets/cli/dist/cli.cjs.dev.js
index 5b0a45b..9c4d853 100644
--- a/node_modules/@changesets/cli/dist/cli.cjs.dev.js
+++ b/node_modules/@changesets/cli/dist/cli.cjs.dev.js
@@ -1173,7 +1173,8 @@ async function getStatus(cwd, {
   if (changedPackages.length > 0 && changesets.length === 0) {
     logger.error("Some packages have been changed but no changesets were found. Run `changeset add` to resolve this error.");
     logger.error("If this change doesn't need a release, run `changeset add --empty`.");
-    process.exit(1);
+    // Prevent CI failures in Renovate PRs that immedeately follow ‘Version Packages’
+    // process.exit(1);
   }
 
   if (output) {
diff --git a/node_modules/@changesets/cli/dist/cli.cjs.prod.js b/node_modules/@changesets/cli/dist/cli.cjs.prod.js
index ba1899b..4138f00 100644
--- a/node_modules/@changesets/cli/dist/cli.cjs.prod.js
+++ b/node_modules/@changesets/cli/dist/cli.cjs.prod.js
@@ -632,7 +632,8 @@ async function getStatus(cwd, {sinceMaster: sinceMaster, since: since, verbose:
     ref: sinceBranch || config.baseBranch
   })).length > 0 && 0 === changesets.length && (logger.error("Some packages have been changed but no changesets were found. Run `changeset add` to resolve this error."), 
   logger.error("If this change doesn't need a release, run `changeset add --empty`."), 
-  process.exit(1)), output) return void await fs__default.default.writeFile(path__default.default.join(cwd, output), JSON.stringify(releasePlan, void 0, 2));
+  // Prevent CI failures in Renovate PRs that immedeately follow ‘Version Packages’
+  /* process.exit(1) */ false), output) return void await fs__default.default.writeFile(path__default.default.join(cwd, output), JSON.stringify(releasePlan, void 0, 2));
   const print = verbose ? verbosePrint : SimplePrint;
   return print("patch", releases), logger.log("---"), print("minor", releases), logger.log("---"), 
   print("major", releases), releasePlan;
diff --git a/node_modules/@changesets/cli/dist/cli.esm.js b/node_modules/@changesets/cli/dist/cli.esm.js
index 5eca136..630028b 100644
--- a/node_modules/@changesets/cli/dist/cli.esm.js
+++ b/node_modules/@changesets/cli/dist/cli.esm.js
@@ -1150,7 +1150,8 @@ async function getStatus(cwd, {
   if (changedPackages.length > 0 && changesets.length === 0) {
     error("Some packages have been changed but no changesets were found. Run `changeset add` to resolve this error.");
     error("If this change doesn't need a release, run `changeset add --empty`.");
-    process.exit(1);
+    // Prevent CI failures in Renovate PRs that immedeately follow ‘Version Packages’
+    // process.exit(1);
   }
 
   if (output) {
