diff --git a/node_modules/@changesets/git/dist/git.cjs.dev.js b/node_modules/@changesets/git/dist/git.cjs.dev.js
index 40cfe7b..cce4c29 100644
--- a/node_modules/@changesets/git/dist/git.cjs.dev.js
+++ b/node_modules/@changesets/git/dist/git.cjs.dev.js
@@ -247,6 +247,10 @@ async function getChangedPackagesSinceRef({
     const prevPkg = fileToPackage[fileName] || {
       dir: ""
     };
+
+    // Fixing "Some packages have been changed but no changesets were found" for private packages
+    if (pkg.packageJson.private) return;
+
     if (pkg.dir.length > prevPkg.dir.length) fileToPackage[fileName] = pkg;
   }));
   return Object.values(fileToPackage) // filter, so that we have only unique packages
diff --git a/node_modules/@changesets/git/dist/git.cjs.prod.js b/node_modules/@changesets/git/dist/git.cjs.prod.js
index 711c0cb..f7756db 100644
--- a/node_modules/@changesets/git/dist/git.cjs.prod.js
+++ b/node_modules/@changesets/git/dist/git.cjs.prod.js
@@ -139,6 +139,7 @@ async function getChangedChangesetFilesSinceRef({cwd: cwd, ref: ref}) {
 }
 
 async function getChangedPackagesSinceRef({cwd: cwd, ref: ref}) {
+  
   const changedFiles = await getChangedFilesSince({
     ref: ref,
     cwd: cwd,
@@ -151,6 +152,10 @@ async function getChangedPackagesSinceRef({cwd: cwd, ref: ref}) {
       const prevPkg = fileToPackage[fileName] || {
         dir: ""
       };
+
+    // Fixing "Some packages have been changed but no changesets were found" for private packages
+    if (pkg.packageJson.private) return;
+
       pkg.dir.length > prevPkg.dir.length && (fileToPackage[fileName] = pkg);
     }));
     var dir;
diff --git a/node_modules/@changesets/git/dist/git.esm.js b/node_modules/@changesets/git/dist/git.esm.js
index 0cc847f..451ebbc 100644
--- a/node_modules/@changesets/git/dist/git.esm.js
+++ b/node_modules/@changesets/git/dist/git.esm.js
@@ -236,6 +236,10 @@ async function getChangedPackagesSinceRef({
     const prevPkg = fileToPackage[fileName] || {
       dir: ""
     };
+
+    // Fixing "Some packages have been changed but no changesets were found" for private packages
+    if (pkg.packageJson.private) return;
+
     if (pkg.dir.length > prevPkg.dir.length) fileToPackage[fileName] = pkg;
   }));
   return Object.values(fileToPackage) // filter, so that we have only unique packages
