diff --git a/node_modules/ajv/dist/core.js b/node_modules/ajv/dist/core.js
index dbc417b..112f165 100644
--- a/node_modules/ajv/dist/core.js
+++ b/node_modules/ajv/dist/core.js
@@ -549,8 +549,10 @@ function checkKeyword(keyword, def) {
     (0, util_1.eachItem)(keyword, (kwd) => {
         if (RULES.keywords[kwd])
             throw new Error(`Keyword ${kwd} is already defined`);
-        if (!KEYWORD_NAME.test(kwd))
-            throw new Error(`Keyword ${kwd} has invalid name`);
+        // commenting out to allow '@context' as a keyword - see apps/site/src/lib/jsonSchema.ts
+        // @todo fix this properly
+        // if (!KEYWORD_NAME.test(kwd))
+        //     throw new Error(`Keyword ${kwd} has invalid name`);
     });
     if (!def)
         return;
