diff --git a/node_modules/playwright-test-coverage/index.js b/node_modules/playwright-test-coverage/index.js
index f02de8d..2417887 100644
--- a/node_modules/playwright-test-coverage/index.js
+++ b/node_modules/playwright-test-coverage/index.js
@@ -3,7 +3,9 @@ const path = require("path");
 const crypto = require("crypto");
 const baseTest = require("@playwright/test").test;
 
-const istanbulCLIOutput = path.join(process.cwd(), ".nyc_output");
+const istanbulCLIOutput = process.env.ISTANBUL_CLI_OUTPUT
+  ? path.resolve(process.env.ISTANBUL_CLI_OUTPUT)
+  : path.join(process.cwd(), ".nyc_output");
 
 function generateUUID() {
   return crypto.randomBytes(16).toString("hex");
