diff --git a/node_modules/playwright-core/lib/server/fetch.js b/node_modules/playwright-core/lib/server/fetch.js
index 06ea087..46d4cd5 100644
--- a/node_modules/playwright-core/lib/server/fetch.js
+++ b/node_modules/playwright-core/lib/server/fetch.js
@@ -186,7 +186,9 @@ class APIRequestContext extends _instrumentation.SdkObject {
       const cookie = parseCookie(header);
       if (!cookie) continue; // https://datatracker.ietf.org/doc/html/rfc6265#section-5.2.3
 
-      if (!cookie.domain) cookie.domain = url.hostname;else (0, _utils.assert)(cookie.domain.startsWith('.'));
+      // This hack allows setting cookies in apps/site/tests/shared/nav.ts → login
+      // To be removed in https://app.asana.com/0/1202542409311090/1202652337622563/f
+      if (!cookie.domain) cookie.domain = url.hostname;else cookie.domain = cookie.domain.replace(/^./, '');
       if (!(0, _cookieStore.domainMatches)(url.hostname, cookie.domain)) continue; // https://datatracker.ietf.org/doc/html/rfc6265#section-5.2.4
 
       if (!cookie.path || !cookie.path.startsWith('/')) cookie.path = defaultPath;
